old_tests/test_reporters.py

```diff
@@ -89,10 +102,10 @@ def test_HDF5Reporter(tmpdir):
     hdf5_reporter = HDF5Reporter(
         flow=flow,
         collision=collision,
-        interval=1,
-        filebase= tmpdir / "output")
+        interval=step,
+        filebase=tmpdir / "output")
     simulation.reporters.append(hdf5_reporter)
-    simulation.step(3)
+    simulation.step(step)
```

Comment:
`simulation(step)` since the step method is deprecated

Reply:
this will be adapted as part of #213